po/git-novice.ja.po

```diff
@@ -4384,31 +4365,31 @@ msgid ""
 "> 5. Save and close"
 msgstr ""
 ">\n"
-"> Jennifer is collaborating on her Python script with her colleagues and\n"
-"> realizes her last commit to the group repository is wrong and wants to\n"
-"> undo it.  Jennifer needs to undo correctly so everyone in the group\n"
-"> repository gets the correct change.  `git revert [wrong commit ID]`\n"
-"> will make a new commit that undoes Jennifer's previous wrong\n"
-"> commit. Therefore `git revert` is different than `git checkout [commit\n"
-"> ID]` because `checkout` is for local changes not committed to the\n"
-"> group repository.  Below are the right steps and explanations for\n"
-"> Jennifer to use `git revert`, what is the missing command?\n"
+"> ジェニファーは同僚とPythonスクリプトで共同作業を行っており、\n"
+"> グループのリポジトリへの彼女の最新のコミットが間違っていることに気付き、\n"
+"> それを元に戻したいと思っています。Jenniferは、グループリポジトリのみんなが正しい変更を取得できるように、\n"
+"> 正しく元に戻す必要があります。  `git revert [wrong commit ID]` は、\n"
+"> ジェニファーの誤ったコミットを元に戻す新しいコミットを作ります。\n"
+"> 従って `git revert` は `git checkout [commit\n"
+"> ID]` とは異なります。なぜなら `checkout` はグループのリポジトリにはコミットされていない\n"
+"> ローカルの変更用のコマンドだからです。以下は、ジェニファーが\n"
+"> `git revert` を使用するための正しい手順と説明ですが、不足しているコマンドは何でしょうか?\n"
 ">\n"
-"> 1. `________ # Look at the git history of the project to find the commit "
-"ID`\n"
+"> 1. `________ # コミットIDを見つけるために、プロジェクトのgitの "
+"履歴を見ます`\n"
 ">\n"
-"> 2. Copy the ID (the first few characters of the ID, e.g. 0b1d055).\n"
+"> 2. そのIDをコピーします (IDの最初の数文字は例えば 0b1d055)。\n"
 ">\n"
 "> 3. `git revert [commit ID]`\n"
 ">\n"
-"> 4. Type in the new commit message.\n"
+"> 4. 新しいコミットメッセージを入力します。\n"
 ">\n"
-"> 5. Save and close"
+"> 5. 保存して閉じます"
 
```

Comment:
This is odd.. it's not a problem with the translation, but I can't find the answer to the quiz question here (or the one before it).

Reply:
Good point.
I also can't find the "Solution" (== the answer)...
https://swcarpentry.github.io/git-novice/05-history/index.html#solution-1
What should we do...